iOS/MobileStarter/react-app/src/HubScreenButton.tsx

```diff
@@ -0,0 +1,28 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import classnames from "classnames";
+import { LoadingSpinner } from "@itwin/core-react";
+import { Button, ButtonProps } from "./Exports";
+
+export function HubScreenButton(props: ButtonProps) {
+  const { className, ...others } = props;
+  return <Button className={classnames("imodel-row", className)} {...others} />;
```

Comment:
Even thought classname is leftover from my code, I don't think it's an appropriate classname for a HubScreenButton.

Reply:
Changed to hubscreen-button